Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -123,52 +135,73 @@ volcano_plot <- function(lmResultsObj,
 ```
 
 ```{r}
-
-lmResults <- run_limma(data_to_model, design, coefficient='materialRBP:toxicToxic')
+limma_fit <- run_limma(data_to_model, design)
```

Comment:
limma fitting now separated into two steps, so that `limma_fit` object is available later.